src/main/app/claims/newFeaturesConsentMiddleware.ts

```diff
@@ -0,0 +1,25 @@
+import * as express from 'express'
+
+import { User } from 'idam/user'
+import { ClaimStoreClient } from 'claims/claimStoreClient'
+import { Paths as ClaimPaths } from 'claim/paths'
+import { FeatureToggles } from 'utils/featureToggles'
+
+export class NewFeaturesConsentMiddleware {
```

Comment:
Please move that file to `src/main/features/claim/guards` as this is closely related to claim issue journey only and it is strongly coupled with 'claim/paths' at the moment.

It could be made more generic so that redirect is configurable but I guess that a bit more work.